src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -43,22 +39,22 @@ const UserInformationContent: React.FunctionComponent<PropsType> = ({
     history.push(`/venue/${venue?.id}`);
   };
 
-  if (!user) return <></>;
+  if (!user || !profile) return <></>;
```

Comment:
The sign up flow is sequenced in two parts: first the user creates an account, then he enters his profile information when he wants to enter the bar.
It means that the Profile Modal should show information when the user is logged in but has no profile information

Reply:
good point

Reply:
@antoineeudes All the information below is from the profile. What's displayed if the user does have one?